test.sh

```diff
@@ -86,6 +86,7 @@ function run_unit_tests() {
     # spuriously because one test is modifying a table (especially
     # registrations) while another test is reading it.
     # https://github.com/letsencrypt/boulder/issues/1499
+    #go test -v "${UNIT_PACKAGES[@]}" "${FILTER[@]}"
```

Comment:
Looks like you left behind a comment?

Reply:
I'm trying to fix this test still. I should have marked this as draft.